### PR TITLE
Grafana dashboard doc draft

### PR DESCRIPTION
**Pull request by DifferentialOrange**
Closes https://github.com/tarantool/grafana-dashboard/issues/45

Add documentation section on Grafana dashboard and set up guide.

### CODE REVIEW

File: doc/monitoring/getting_started.rst [hunk 1]
```diff
@@ -194,3 +194,120 @@ via configuration.
...
+
+    ..  image:: images/grafana-import-v6.png
+        :align: center
+
+    To import specific dashboard, choose one of the following options:
+
+    - paste dashboard id (``12567`` for InfluxDB dashboard, ``13054`` for Prometheus dashboard), or
+    - paste link to dashboard (https://grafana.com/grafana/dashboards/12567 for InfluxDB dashboard,
+      https://grafana.com/grafana/dashboards/13054 for Prometheus dashboard), or
+    - paste dashboard json file contents, or
```
Comment (vasiliy-t) [inline doc/monitoring/getting_started.rst:302]: is it possible to provide direct link to download json file?

**Review by ZolotarevIgor-mail-ru — CHANGES_REQUESTED** [review]
Please add a pic "how Grafana dashboard is look like"

File: doc/monitoring/getting_started.rst (hunk 1 above)
Comment (Onvember) [reply doc/monitoring/getting_started.rst:302]: It seems to be possible, according to the Sphinx docs ([:download: role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#referencing-downloadable-files)), but we never tried

Comment (DifferentialOrange) [reply doc/monitoring/getting_started.rst:302]: It is possible (e.g. here https://github.com/tarantool/grafana-dashboard/releases/tag/0.2.2) to make a link to specific revision, so we will need to update it every time I upload a new version.

**Comment by DifferentialOrange** [comment]
Closed in favor of https://github.com/tarantool/metrics/pull/185 , https://github.com/tarantool/doc/pull/1802 , https://github.com/tarantool/grafana-dashboard/pull/50

**Pull request closed without merging**